test/common/network/connection_impl_test.cc

```diff
@@ -2880,6 +2880,19 @@ TEST_F(PostCloseConnectionImplTest, NoReadAfterCloseFlushWriteWriteData) {
   file_ready_cb_(Event::FileReadyType::Read);
 }
 
+// Test that close(ConnectionCloseType::Abort) won't write and flush pending data.
+TEST_F(PostCloseConnectionImplTest, CloseAbort) {
+  InSequence s;
+  initialize();
+  writeSomeData();
+
+  // Connection abort. We have data written above in writeSomeData(),
+  // it won't be written and flushed due to Abort.
```

Comment:
super nit, s/Abort/abort

Reply:
hmm `abort` might fail grammar  in some sense. I use ``ConnectionCloseType::Abort`` if you don't mind. Thanks for catching this!